src/MongoDB.Driver.Core/Core/Operations/CreateIndexesOperation.cs

```diff
@@ -155,7 +155,9 @@ internal IWriteOperation<BsonDocument> CreateOperation(SemanticVersion serverVer
             }
             else
             {
+#pragma warning disable 618
                 return new CreateIndexesUsingInsertOperation(_collectionNamespace, _requests, _messageEncoderSettings);
```

Comment:
Cannot remove this option because the alternative option (line 148) `Feature.CreateIndexesCommand` starts support from 3.0 server version.

Reply:
Why is the pragma new?

Reply:
Our C# code that says CreateIndexesCommand was first supported in server 3.0 might be wrong.

Server documentation says it is new in server version 2.6:

https://docs.mongodb.com/v2.6/reference/command/createIndexes/